admin_manual/installation/source_installation.rst

```diff
@@ -277,6 +277,32 @@ Keep in mind that changes to php.ini may have to be done on more than one ini fi
 
   /etc/php5/cli/php.ini
 
+**Notes when using php-fpm:**
+
+It may be the case when using ``php-fpm``, system environment variables like PATH, TMP or others are not automatically 
+populated in the same way compared when using ``php-cli``.
+
+A php call like ``getenv('PATH');`` can therefore return an empty result.
+Even fallback content for those variables is most times defined, this may not match 100% of the needs. 
+In those cases, the variables need to be set manually in the corresponding ``php-fpm`` ini file. 
+For Ubuntu, the location for these ini files can be found at: ``/etc/php5/fpm/``
+
+Usually, you will find some or all of this environment variables beeing commented out like:
+::
+
+	;env[HOSTNAME] = $HOSTNAME
+	;env[PATH] = /usr/local/bin:/usr/bin:/bin
+	;env[TMP] = /tmp
+	;env[TMPDIR] = /tmp
+	;env[TEMP] = /tmp
+
+You can either just uncomment the existing one and/or copy the output from the command shell eg: ``printenv PATH`` to 
+the environment variable to be set. If no system environment variable is present to be uncommented, 
+you need to create one.
+
+Please keep in mind, that it is possible to create different settings for ``php-cli`` and ``php-fpm``
+and even for different domains/websites. Please take care to take the right one. Your result can be tested with ``phpinfo()``
+which must accessed via a browser to your domain/website.
```

Comment:
Line breakes are automatically corrected. That´s just a presentation thing of the raw data, final is fine. I checked it in my browser. Please correct me if you experience differences when you test this PR.

Nobody can say in advance which one applies to owncloud. This is the last step the admin has to decide on his own. This is also the reason why I added the examples with ``;env[]`` so the admin can search for.